Live-Snap/Live-Snap/Controllers/FetchSnapcodeViewController.swift

```diff
@@ -158,16 +160,21 @@ class FetchSnapcodeViewController: UIViewController, UITextFieldDelegate {
             return nil
         }
         
-        if result.count < 3 {
+        if result.count < 3 || result.count > 15 {
```

Comment:
lets make it result.count > 30 in case snap increases the username character limit